### PR TITLE
add FAQs and update glossary

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: docs/partials/_faq-partial.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+<p></p>
+
+<p>For more, see "<a href="https://docs.arbitrum.foundation/fee-distribution">fee distribution</a>."</p>
+
+<p></p>
+
+
+
+### Will there ever be more than the two current Arbitrum DAO governed chains (Arbitrum DAO and Nova)? {#will-there-ever-be-more-than-the-two-current-arbitrum-dao-governed-chains-arbitrum-dao-and-nova}
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:30]: todo on Notion: `(Arbitrum DAO` -> `(Arbitrum One`

File: docs/partials/_faq-partial.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+<p>The <a href="/dao-glossary#core-governor">Core Governor</a> handles <a href="/dao-glossary#constitutional-aip">Constitutional AIP</a>s; these can modify core, critical parts of the system, and are executed only after a delay, ensuring users have a window to withdraw before they effectuate. The <a href="/dao-glossary#treasury-governor">Treasury Governor</a> handles non-constitutional changes, like distributing funds out of the <a href="/dao-glossary#arbitrum-dao-treasury">Arbitrum DAO Treasury</a>; these are effectuated faster than core governance proposals.</p>
+
+
+
+### Why can't the Arbitrum DAO use social forms of governance like Ethereum? Why does L2 governance need to be on-chain? {#why-cant-the-arbitrum-dao-use-social-forms-of-governance-like-ethereum-why-does-l2-governance-need-to-be-onchain}
+<p>Layer 1 chain like Ethereum are ultimately governed by social consensus. L2's like Arbitrum One, by contrast, derive their security from smart contracts on L1; thus, changes to the core system need to made explicitly via on-chain smart contract upgrade. See "<a href="https://docs.arbitrum.foundation/why-governance">Why Governance</a>" for more.</p>
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:43]: nits todo on Notion: `Layer 1 chain` -> `Layer 1 chains`, `L2's` -> `L2s`

File: docs/partials/_faq-partial.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+### What does it mean for an ARB voter to "exclude" their votes? Why is this functionality beneficial?   {#what-does-it-mean-for-an-arb-voter-to-exclude-their-votes-why-is-this-functionality-beneficial--}
+<p>An $ARB holder can exclude their votes by delegating to a special exclude address (<a href="https://arbiscan.io/address/0x00000000000000000000000000000000000a4b86">0x00000000000000000000000000000000000a4b86</a>). This means their tokens' votes count in the denominator for <a href="/dao-glossary#quorum">Quorum</a>  calculation (5% of votable tokens for the <a href="/dao-glossary#core-governor">Core Governor</a>, 3% for the <a href="/dao-glossary#treasury-governor">Treasury Governor</a>). This is useful to prevent "artificially" inflating the quorum requirement for token holder that can't or don't plan on voting. The primary user case for the exclude address is the <a href="/dao-glossary#arbitrum-dao-treasury">Arbitrum DAO Treasury</a>, which is currently the largest single holder of ARB and is delegated to the exclude address, but any address capable of delegating can opt to delegate to the exclude address as well.</p>
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:67]: nit todo on Notion (possible brainfart on my end): `votes count` -> `votes won't count`

File: docs/partials/_faq-partial.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+<p></p>
+
+
+
+## $ARB token and airdrop
+
+### When does the $ARB airdrop end? {#when-does-the-arb-airdrop-end}
+<p>The $ARB airdrop ends at L1 Ethereum block <a href="https://etherscan.io/block/countdown/18208000">18208000</a> (estimated to be 9/24/23.</p>
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:76]: nit todo on Notion: `/23.` -> `/23).`

File: docs/partials/_faq-partial.md (hunk 3 above)
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:67]: `largest single holder of ARB` -> `largest single holder of $ARB`

File: docs/partials/_faq-partial.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+
+
+### What happens to all of the ARB tokens that don't get claimed in the airdrop? {#what-happens-to-all-of-the-arb-tokens-that-dont-get-claimed-in-the-airdrop}
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:80]: `all of the ARB` -> `all of the $ARB`

File: docs/partials/_faq-partial.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+<p>Members of the <a href="/dao-glossary#security-council">Security Council</a> are split into two six-member cohorts, one of which is up for on-chain election every six months.</p>
+
+<p>See <a href="https://arbitrumfoundation.medium.com/security-council-elections-101-3cbdee20ca82">here</a> for more info.</p>
+
+
+
+### What's the difference between Security Council Emergency Actions and Non-Emergency Actions? {#whats-the-difference-between-security-council-emergency-actions-and-nonemergency-actions}
+<p><a href="/dao-glossary#emergency-action">Emergency action</a>s are actions taken by the <a href="/dao-glossary#security-council">Security Council</a> in response to a security emergency, such as a critical vulnerability that could significantly compromise the integrity, confidentiality, or availability of a chain governed by the <a href="/dao-glossary#arbitrum-dao">Arbitrum DAO</a>. They require 9 of the 12 Security Council members to sign, and the are executed immediately.  At the appropriate time, are to be followed by a <a href="/dao-glossary#transparency-report">Transparency report</a>. </p>
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:124]: nit for Notion: `time, are` -> time, Emergency actions are`

File: docs/partials/_faq-partial.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+### What are the requirements for being a delegate? {#what-are-the-requirements-for-being-a-delegate}
+<p>There are none (technically)! You just have to own <a href="/dao-glossary#arb">$ARB</a> and/or appeal to the community on why they should choose you to represent them when voting on proposals in governance.</p>
+
+### How do I become a delegate? {#how-do-i-become-a-delegate}
+<p>An Arbitrum DAO delegate is anyone to whom an <a href="/dao-glossary#arb">$ARB</a> holder has delegated votes. So to become a delegate, you can either get an $ARB holder to delegate their votes to you, or, if you hold $ARB, you can delegate votes to yourself. </p>
+
+
+
+### Can I delegate my voting power to more than one delegate? {#can-i-delegate-my-voting-power-to-more-than-one-delegate}
+<p>Yes; voting power can be split between any number of delegtes.</p>
```
Comment (symbolpunk) [inline docs/partials/_faq-partial.md:199]: nit for notion: `delegtes` -> `delegates`

**Review by symbolpunk — CHANGES_REQUESTED** [review]
This is awesome - I didn't use the "commit suggestions" for the nits because changes will have to be made in faq cms (happy to update Notion and regen within your branch if you'd prefer)

**Comment by DZGoldman** [comment]
> happy to update Notion and regen within your branch if you'd prefer

no this is perfect, I'll make the updates on notion and push, thanks!

**Review by symbolpunk — APPROVED** [review]

**Comment by anhlam2234** [comment]
>

**Pull request merged**